Sources/SKSupport/LoggableMessageTypes.swift

```diff
@@ -85,33 +30,51 @@ fileprivate extension Encodable {
   }
 }
 
-extension Request: CustomStringConvertible, CustomLogStringConvertible {
+// MARK: - DocumentURI
+
+extension DocumentURI: CustomLogStringConvertible {
+  public var redactedDescription: String {
+    return "<DocumentURI length=\(description.count) hash=\(description.hashForLogging)>"
+  }
+}
+
+// MARK: - RequestType
+
+fileprivate struct AnyRequestType: CustomLogStringConvertible {
```

Comment:
FWIW SE-0364 will eventually require these to be marked `@retroactive` since they're conformances of external types to external protocols

Reply:
Yeah, I need to address these warnings across the entire codebase. I’ll probably create a PR in the next week or so when I get annoyed by the warnings again.